docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,62 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into the code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
+the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>`.
+
+Before opening a pull request
+=============================
+
+Before you create the pull request, you should go through the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>` to ensure
+the proposed changes meet the repository standards.
+
+The PR should be as small as possible. However, the code included in the PR should be complete. It should complete an entire feature, but this doesn't necessarily mean it completes an entire issue.
+However, all code merged into the ``dev`` branch should meet the repository standards and work as expected.
+
+If you want to work on the pull request or are not yet finished with the code, please indicate this by marking the pull request as a
+`draft or WIP <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_ PR.
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
```

Comment:
I feel like draft PR means it's not in final state but would like someone to give initial review.

Reply:
yes. Thank you!